Sources/Auth/AuthClientFactory.swift

```diff
@@ -6,10 +6,11 @@ public struct AuthClientFactory {
         projectId: String,
         crypto: CryptoProvider,
         networkingClient: NetworkingInteractor,
-        pairingRegisterer: PairingRegisterer
+        pairingRegisterer: PairingRegisterer,
+        groupIdentifier: String
     ) -> AuthClient {
         let logger = ConsoleLogger(loggingLevel: .off)
-        let keyValueStorage = UserDefaults.standard
+        let keyValueStorage = UserDefaults(suiteName: groupIdentifier) ?? UserDefaults.standard
```

Comment:
@flypaper0 what do you think of fallback to UserDefaults .standard?

Reply:
with fallback we're give developers a chance to make mistake. I'd prefer fatal error here

Reply:
check NotifyClientFactory line 104